static/app/views/issueList/actions/actionSet.tsx

```diff
@@ -170,6 +171,34 @@ function ActionSet({
         });
       },
     },
+    ...(organization.features.includes('issue-priority-ui')
+      ? [
+          {
+            key: 'set-priority',
+            label: t('Set Priority to...'),
+            hidden: !organization.features.includes('issue-priority-ui'),
+            isSubmenu: true,
+            children: [PriorityLevel.HIGH, PriorityLevel.MEDIUM, PriorityLevel.LOW].map(
+              priority => ({
+                key: `priority-${priority}`,
+                textValue: t('Set priority to %s', priority),
+                label: <GroupPriorityBadge priority={priority} />,
+                onAction: () =>
+                  openConfirmModal({
+                    bypass: !onShouldConfirm(ConfirmAction.SET_PRIORITY),
+                    onConfirm: () => onUpdate({priority}),
+                    message: confirm({
+                      action: ConfirmAction.SET_PRIORITY,
+                      append: ` to ${priority}`,
+                      canBeUndone: true,
```

Comment:
can it be undone?

Reply:
Hmm so I think the logic for the undo stuff is in overview.tsx. I'll add that in another PR. Either way this isn't something that can't be undone like merge and delete so I wouldn't want to show that message in the confirm modal